frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/mocks/applicationMetadataMock.ts

```diff
@@ -34,9 +35,16 @@ const mockOnEntry: OnEntry = {
   show: 'select-instance',
 };
 
+const mockDataTypes: DataTypeElement[] = [
+  {
+    id: 'dataTypeId',
```

Comment:
Maybe this could be defined as a const and exported as well? So we can use the direct reference in the tests?